### PR TITLE
Feature/add style option

### PR DESCRIPTION
**Pull request by bgilhome-lds**
Adds a prop 'addStyle' (default true), to control adding the base styles (grey border, padding etc).

### CODE REVIEW

**Comment by metonym** [comment]
What is your current workaround?

**Comment by bgilhome-lds** [comment]
Hi @metonym, my current workaround is to use !important on all styles that I need to override the packaged styles.

File: src/AccordionItem.svelte [hunk 1]
```diff
@@ -27,12 +28,12 @@
 </script>
 
 <style>
-  li {
+  .default li {
```
Comment (metonym) [inline src/AccordionItem.svelte:31]:
```suggestion
  li.default {
```

".default li" is unused

File: src/AccordionItem.svelte [hunk 2]
```diff
@@ -4,6 +4,7 @@
   export let expanded = false;
   export let disabled = false;
   export let buttonClass = "";
+  export let style = "default";
```
Comment (metonym) [inline src/AccordionItem.svelte:7]: Calling the class name "style" may be misleading. Also, it could be overridden by `$$restProps.style`.

**Comment by metonym** [comment]
@bgilhome-lds understood. However, `$$restProps` is already forwarded to the `li` element.

Why not do `<AccordionItem class="custom-class">`?

**Comment by bgilhome-lds** [comment]
@metonym Ah, yes that works. Thanks, this can be closed!

**Comment by bgilhome-lds** [comment]
Actually, the packaged AccordionItem styles are still overriding any custom styles unless I use !important. Are there any thoughts against adding an addStyle (or other named) option?

**Comment by metonym** [comment]
What about passing the class name to the parent `Accordion` to increase the CSS selector specificity?

**Comment by metonym** [comment]
A last resort would be to export a completely unstyled version of `AccordionItem`

**Pull request closed without merging**